src/scripts/main.js

```diff
@@ -1,3 +1,63 @@
 'use strict';
 
-// write code here
+const appendRowButton = document.querySelector('.append-row');
+const removeRowButton = document.querySelector('.remove-row');
+const appendColumnButton = document.querySelector('.append-column');
+const removeColumnButton = document.querySelector('.remove-column');
+const tableBody = document.querySelector('.field > tbody');
+
+appendColumnButton.addEventListener('click', (e) => {
+  const rows = [...document.querySelector('.field > tbody').children];
+
+  if (removeColumnButton.disabled) {
+    removeColumnButton.disabled = false;
+  }
+
+  if (rows[0].cells.length === 9) {
+    e.target.disabled = true;
+  };
+
```

Comment:
Also you can create 1 function where you check if the button should be disabled, and use it in all the event listeners